src/api/kube.ts

```diff
@@ -54,7 +61,7 @@ export class KubeHelper {
   }
 
   async deleteAllServices(namespace = '') {
-    const k8sApi = this.kc.makeApiClient(CoreV1Api)
+    const k8sApi = KubeHelper.KUBE_CONFIG.makeApiClient(CoreV1Api)
```

Comment:
What if we need to have different configs? For example make queries from different accounts?

Reply:
I have not faced such a case

Reply:
But we may face it one day...

Reply:
Could you elaborate more?

Reply:
I mean not this specific line of code, I am talking about more general stuff. Maybe we do not need it at all (as we don't right now), but static kubeconfig makes chectl somewhat limited.

Reply:
It is absolutely not true, it brings consistency. Image the case when `chectl` is working and meanwhile user changed the context? Then a new instance of KubeHelper will use a kubeconfig with a different context that might lead to errors.